### PR TITLE
<AddItem /> - added possibility to show tooltip for all themes

### PR DESCRIPTION
**Pull request by egidijusja**
### 🔦 Summary

...

### ✅ Checklist

- [ ] 👨‍💻 API change is approved by the UI Infra Developers 
- [ ] 👨‍🎨 UX change is approved by the Design System UX 
- 📚 Change is documented
  - [ ] Story
  - [ ] API description
  - [ ] Cheatsheet
  - [ ] Other (explain)
- 🔬 Change is tested
  - [ ] Component
  - [ ] Visual test

### CODE REVIEW

**Comment by WixBuildServer** [comment]
### 🤖 Hi, I'm #Janet!

  A deploy preview of your PR was successfully released 🖼.

  You can find it under one of the following URLs:
  * https://preview.wix-style-react.com/pr_6666

**Comment by EgoziE** [comment]
Can you please explain the scope of this PR?

**Comment by egidijusja** [comment]
> Can you please explain the scope of this PR?
> AddItem has tooltip already.

Currently tooltip can be shown only when theme is `image`. I'm working on `showTitle` feature. We might need to show tooltip for other themes as well. Unfortunately there are some questions left, so we will have a meeting with UX guys. Made this PR as a draft.

**Comment by egidijusja** [comment]
Hello @milkyfruit, we have added possibility to show tooltip for every theme on AddItem component as discussed earlier. Also added new example section `Title` with two examples:
1) Component with title.
2) Component with tooltip instead of title.

Right now there is possibility to have title and tooltip both at the same time on `AddItem` component. Should we add this example as well? Also can you please review this example in general (title, description, etc)

File: packages/wix-style-react/src/AddItem/docs/index.story.js [hunk 1]
```diff
@@ -89,6 +89,12 @@ export default {
             source: examples.plain,
           }),
 
+          example({
+            title: 'Title',
+            text: 'AddItem title options',
```
Comment (milkyfruit) [inline packages/wix-style-react/src/AddItem/docs/index.story.js:94]: @egidijusja please change text to: "AddItem title can be displayed as visible always or hidden in a tooltip. It's controlled with `children` and `tooltipContent` props."

**Review by milkyfruit — CHANGES_REQUESTED** [review]
@egidijusja 

- [ ] please change the description text.
- [ ] When alignItems is set to "left" or "right" then tooltip's arrow appears in the opposite side that it should be:
image

Unfortunately, I couldn't reproduce the issue when using tooltip in a playground with the same props: https://www.wix-style-react.com/?path=/story/introduction-playground--playground&snippet=7cecc445-faf8-40c8-8b5f-2cbcd2cd9d40

**Comment by milkyfruit** [comment]
> Hello @milkyfruit, we have added possibility to show tooltip for every theme on AddItem component as discussed earlier. Also added new example section `Title` with two examples:
> 
> 1. Component with title.
> 2. Component with tooltip instead of title.
> 
> Right now there is possibility to have title and tooltip both at the same time on `AddItem` component. Should we add this example as well? Also can you please review this example in general (title, description, etc)

No need to show example with title and tooltip at once, it's an edge case.

**Review by mykas — APPROVED** [review]
Nice work!

**Pull request merged**